metallb/run_e2e.sh

```diff
@@ -15,16 +18,15 @@ sudo firewall-cmd --zone=libvirt --permanent --add-port=180/tcp
 sudo firewall-cmd --zone=libvirt --add-port=180/tcp
 
 SKIP="none"
-if [ "${IP_STACK}" = "v4" ]; then SKIP="${SKIP}\|IPV6"; fi
-if [ "${IP_STACK}" = "dual" ]; then SKIP="${SKIP}\|IPV6"; fi
-if [ "${IP_STACK}" = "v6" ]; then SKIP="${SKIP}\|IPV4\|BGP"; fi
+if [ "${IP_STACK}" = "v4" ]; then SKIP="${SKIP}\|IPV6" && export PROVISIONING_HOST_EXTERNAL_IPV4=${PROVISIONING_HOST_EXTERNAL_IP}; fi
```

Comment:
You may need to update this with the DUALSTACK tests that are now available upstream. See https://github.com/metallb/metallb/blob/main/.circleci/config.yml#L41